.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,6 +19,7 @@
 - [ ] The PR has an appropriate "CHANGELOG: ..." label (If you are unsure of which label to add, ask the reviewers for guidance).
 - [ ] The PR follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
 - [ ] The PR addresses the points mentioned in the codeowner checks for the files/folders changed. (See the [codeowner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).)
-- [ ] The PR is assigned to an appropriate reviewer.
+- [ ] The PR is **assigned** to an appropriate reviewer.
   - If you're a new contributor, please ask on [Gitter](https://gitter.im/oppia/oppia-chat) for someone to assign a reviewer.
```

Comment:
Maybe add: "and don't tick this checkbox"

Reply:
Done.